.github/workflows/container.yml

```diff
@@ -0,0 +1,34 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Container
+
+on: workflow_dispatch
+
+jobs:
+  container:
+    name: container
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - name: Docker build
+        run: |
+          cd docker
+          docker build . -t skywalking-php
+          docker tag skywalking-php ghcr.io/apache/skywalking-php:latest
+          docker tag skywalking-php ghcr.io/apache/skywalking-php:v0.1.0
```

Comment:
Does this seem like you are building the image repeatedly?

Reply:
I think you should follow tag or release event of GHA?

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Reply:
I want to do it manually, it doesn't seem necessary now